client/lib/add-query-args/README.md

```diff
@@ -0,0 +1,16 @@
+addQueryArgs
```

Comment:
Thanks for pointing out that module. After looking at it, I don't think the query args logic would be a good fit in `path` since that logic seems to be used for Calypso specific site fragment logic.

I'd suggest that we make it `lib/route/addQueryArgs`. Alternatively, it may be useful to eventually add `removeQueryArg. So, in that case, perhaps we create `lib/route/query-args` and then nest `addQueryArgs` under that?